pkg/controllers/dashboard/systemcharts/controller.go

```diff
@@ -120,13 +119,9 @@ func (h *handler) onRepo(key string, repo *catalog.ClusterRepo) (*catalog.Cluste
 		}
 		// webhook needs to be able to adopt the MutatingWebhookConfiguration which originally wasn't a part of the
 		// chart definition, but is now part of the chart definition
-		minVersion := chartDef.MinVersionSetting.Get()
-		exactVersion := chartDef.ExactVersionSetting.Get()
-		if chartDef.ChartName == chart.WebhookChartName && minVersion != "" {
-			exactVersion = ""
-		}
+		version := chartDef.VersionSetting.Get()
 		forceAdopt := chartDef.ChartName == chart.WebhookChartName || chartDef.ChartName == chart.ProvisioningCAPIChartName
-		if err := h.manager.Ensure(chartDef.ReleaseNamespace, chartDef.ChartName, minVersion, exactVersion, values, forceAdopt, installImageOverride); err != nil {
+		if err := h.manager.Ensure(chartDef.ReleaseNamespace, chartDef.ChartName, version, version, values, forceAdopt, installImageOverride); err != nil {
```

Comment:
Min Version should be changed to an empty string.
```suggestion
		if err := h.manager.Ensure(chartDef.ReleaseNamespace, chartDef.ChartName, "", version, values, forceAdopt, installImageOverride); err != nil {
```